xbrowse_server/base/views/family_views.py

```diff
@@ -138,7 +139,7 @@ def edit_family(request, project_id, family_id):
             family.save()
 
             try:
-                seqr_family = get_seqr_family_from_base_family(family)
+                seqr_family = find_matching_seqr_model(family)
                 seqr_family.coded_phenotype = family.coded_phenotype
                 seqr_family.description = family.short_description
                 seqr_family.analysis_notes = family.about_family_content
```

Comment:
I think if you used `update_xbrowse_model` above you wouldn't need to also do this here